qiskit/primitives/base_sampler.py

```diff
@@ -272,6 +292,56 @@ def __call__(
             **run_options,
         )
 
+    def run(
+        self,
+        circuits: Sequence[QuantumCircuit],
+        parameter_values: Sequence[Sequence[float]] | None = None,
+        **run_options,
```

Comment:
Since we allow `parameters` as an input to `__init__`, it should be carried to `run()` as well. For users who want to specify their own order of parameter values to apply.

Reply:
Support in 37f36a6.